fe1-web/src/features/digital-cash/components/TransactionHistory.tsx

```diff
@@ -191,7 +192,8 @@ const TransactionHistory = ({ laoId }: IPropTypes) => {
 };
 
 const propTypes = {
-  laoId: PropTypes.string.isRequired,
+  laoId: PropTypes.oneOfType([PropTypes.instanceOf(Hash).isRequired, PropTypes.string.isRequired])
```

Comment:
It makes me slightly uncomfortable to relax the typing on the properties, but it's good that we're moving everything to tolerate `Hash` as an input.

Reply:
Hmm yes. But I think Hash is probably the way to go for the laoId so I guess we we can also do a refactor that drops support for strings? Should be easier now :)

Reply:
Yes, in principle moving everything to Hash would be nice.

However, I also remember students struggled in the past with Hash's though, due to a couple of things:
1. they're objects, so equality comparisons do not work as expected
2. when rehydrating data from redux, it's easy to forget rehydrating a string into a Hash 

What do you think ?

Reply:
Hmm.. I can definitely see how 1) causes an issue but (unfortunately) this applies to many objects and should probably be learned at some point? I mean it already starts with `({x: 1}) == ({x: 1})`

And for 2) I see the problem too but I guess it should be noticeable if a lot of `new Hash(` appear in the code (assuming we changed the interfaces to only return Hashes) as the other components / functions expect it to be of this type.

I think doing a refactor of this should be doable but I'll do it in a separate PR :)

Reply:
Yes, that it should for sure!